backend/SS_Position/core.py

```diff
@@ -23,6 +26,6 @@ def processMovement(player, key):
 
     if mapWorker.checkPositionMovable(x,y):
         redisWorker.setPlayerPosition(player,x,y)
-        return "MOVE " + str(x) + " " + str(y)
+        return "MOVE " + str(x) + " " + str(y) + " " + door_closed
```

Comment:
This is actually unnecessary according to previous comment. Do not harm already existing functions. Just create your own.